transcom/mymove
# B 19177 int qae/csr

## [B-19177](https://www13.v1host.com/USTRANSCOM38/assetdetail.v1?number=B-19177)

## Summary

This task adds 'Scheduled' before dates in the QAE/CSR Search results table to reflect the status of dates that appear there, as well as reintroducing a couple of tests related to these values.

Is there anything you would like reviewers to give additional scrutiny?

[this article](tbd) explains more about the approach used.

## Verification Steps for the Author

These are to be checked by the author.


## Verification Steps for Reviewers

These are to be checked by a reviewer.


### Setup to Run the Code

- [Instructions for starting storybook](https://transcom.github.io/mymove-docs/docs/frontend/setup/storybook)
- [Instructions for starting the MilMove application](https://transcom.github.io/mymove-docs/docs/getting-started/application-setup/)
- [Instructions for running tests](https://transcom.github.io/mymove-docs/docs/getting-started/development/testing)

### How to test

1. Access the office app
2. Login as a QAE/CSR
3. Search for a Move 
4. Ensure the table headers for Pickup Date and Delivery Date include the word Scheduled, as with the screenshot below: 
<img width="1565" alt="image" src="https://github.com/transcom/mymove/assets/148906585/6e467f1e-6ee5-404f-979f-97495e85e295">
5. Repeat for the Services Counselor's move search with the counselor create move flag enabled. 
6.  Ensure all tests are passing.

### Frontend


### Backend


### Database

#### Any new migrations/schema changes:


## Screenshots
